lib/filter/filter-expr-grammar.ym

```diff
@@ -146,6 +146,7 @@ filter_simple_expr
 	: KW_FACILITY '(' filter_fac_list ')'	{ $$ = filter_facility_new($3);  }
 	| KW_FACILITY '(' LL_NUMBER ')'		{ $$ = filter_facility_new(0x80000000 | $3); }
 	| KW_SEVERITY '(' filter_severity_list ')' { $$ = filter_severity_new($3); }
+  | KW_SEVERITY '(' LL_NUMBER ')'		{ $$ = filter_severity_new( 1 << $3); }
```

Comment:
I toolks like that the indentation here is different than the rest of the file

Reply:
Lately my IDE has problems with identifying the correct indentation style to use. Fixed.